File: data/src/main/java/com/hgianastasio/biblioulbrav2/data/base/historybooks/HistoryBooksOnCache.java
package com.hgianastasio.biblioulbrav2.data.base.historybooks;

import com.hgianastasio.biblioulbrav2.data.disk.GenericCache;
import com.hgianastasio.biblioulbrav2.data.models.historybooks.HistoryBookEntity;

import java.util.List;

/**
 * Created by heitorgianastasio on 4/29/17.
 */

public interface HistoryBooksOnCache extends GenericCache<List<HistoryBookEntity>> {}

File: data/src/main/java/com/hgianastasio/biblioulbrav2/data/base/historybooks/HistoryBooksOnCache.kt
package com.hgianastasio.biblioulbrav2.data.base.historybooks

import com.hgianastasio.biblioulbrav2.data.disk.GenericCache
import com.hgianastasio.biblioulbrav2.data.models.historybooks.HistoryBookEntity

/**
 * Created by heitorgianastasio on 4/29/17.
 */
interface HistoryBooksOnCache : GenericCache<List<HistoryBookEntity>>